addons/mail/static/src/components/follower_list_menu/follower_list_menu.xml

```diff
@@ -5,28 +5,28 @@
         <div class="o_FollowerListMenu" t-attf-class="{{ className }}" t-on-keydown="_onKeydown" t-ref="root">
             <t t-if="thread">
                 <div class="o_FollowerListMenu_followers" t-ref="dropdown">
-                    <button class="o_FollowerListMenu_buttonFollowers btn btn-link" t-att-disabled="props.isDisabled" t-on-click="_onClickFollowersButton" title="Show Followers">
+                    <button class="o_FollowerListMenu_buttonFollowers btn btn-link" t-att-disabled="props.isDisabled or (thread.followers.length === 0 and !thread.hasWriteAccess)" t-on-click="_onClickFollowersButton" title="Show Followers">
```

Comment:
@seb-odoo  Indeed that should be the case. We are hiding the dropdown only when there are no followers `AND` user does not has write access to the record (because there is nothing to show). Even if there's a single follower, the list will be opened. The test cases are also written / adapted for all the cases.

Reply:
@seb-odoo Should be okay now.